platform/conf/conf.go

```diff
@@ -893,7 +1022,7 @@ func keysToStrings(keys []*agent.Key) (keyStrs []string) {
 // Returns false in the case of empty configs as on most platforms,
 // this will default back to cloudconfig
 func (c *Conf) IsIgnition() bool {
-	return c.ignitionV1 != nil || c.ignitionV2 != nil || c.ignitionV21 != nil || c.ignitionV22 != nil || c.ignitionV23 != nil || c.ignitionV3 != nil
+	return c.ignitionV1 != nil || c.ignitionV2 != nil || c.ignitionV21 != nil || c.ignitionV22 != nil || c.ignitionV23 != nil || c.ignitionV3 != nil || c.ignitionV31 != nil || c.ignitionV32 != nil || c.ignitionV33 != nil
```

Comment:
if this here needs to be extended with V31, V32, V33 I think we also have to do it for `AddSystemdUnitDropin`, `AddFile` and `getIgnitionValidateValue`

Reply:
ok like we did for `copyKeys` and `merge` function... will try to see if we can avoid copy paste

Reply:
sure, just wanted to make sure that this is known because otherwise I think at least vmware tests will fail

Reply:
added missing helpers in a following commit.